### PR TITLE
Added dockerfile for building skipper with lightstep integration.

### PR DESCRIPTION
**Pull request by arjunrn**
Added dockerfile and makefile for building skipper with lightstep integration. Also added new step to CDP build pipeline.

### CODE REVIEW

**Comment by arjunrn** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

File: packaging/skipper.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+exec /usr/bin/skipper \
+    -plugindir /plugins \
+"$@"
```
[the remark this answers is not in the document]
Comment (arjunrn) [reply packaging/skipper.sh]: This file is being used as the entrypoint. Or am I missing something?

[the remark this answers is not in the document]
Comment (mikkeloscar) [reply packaging/skipper.sh]: I ment like this:

```
ENTRYPOINT ["/usr/bin/skipper", "-plugindir", "/plugins"]
```

[the remark this answers is not in the document]
Comment (szuecs) [reply packaging/skipper.sh]: Yes it makes sense, but sh+exec does not hurt. It’s pretty much the same and exec doesn’t fork, so we don’t have another process created ;)
So from runtime point of view it’s the same.

[the remark this answers is not in the document]
Comment (mikkeloscar) [reply packaging/skipper.sh]: @szuecs ah true, I forgot what `exec` does :x

I would still argue it's simpler to not have the extra `skipper.sh` file and COPY it into the Docker image :)

[the remark this answers is not in the document]
Comment (szuecs) [reply packaging/skipper.sh]: https://github.com/zalando/skipper/pull/760

**Pull request merged**